public/app/features/alerting/unified/components/rule-viewer/RuleViewer.tsx

```diff
@@ -123,7 +116,6 @@ const RuleViewer = () => {
           {activeTab === ActiveTab.Details && <Details rule={rule} />}
         </TabContent>
       </Stack>
-      {deleteModal}
       {duplicateRuleIdentifier && (
```

Comment:
(Potentially TODO: unify the clone rule logic as well, as this is repeated across two places)